test/System.Net.Http.Formatting.Test/Formatting/BsonMediaTypeFormatterTests.cs

```diff
@@ -340,9 +340,9 @@ public async Task FormatterThrowsOnReadWithInvalidContent()
             stream.Seek(0L, SeekOrigin.Begin);
 
             // Act & Assert
-            await Assert.ThrowsAsync<OverflowException>(
+            await Assert.ThrowsAsync<JsonReaderException>(
```

Comment:
Note to self: Add following to final commit description
- `decimal` overflow now `throw`s a `JsonReaderException`